source/dump.f90

```diff
@@ -847,29 +847,33 @@ subroutine dump_initialise
         ! # particleID turn s[m] x[mm] xp[mrad] y[mm] yp[mrad] z[mm] (E-E0)/E0[1] ktrack
         if(dump_hdf5Format(3) == 0) then
-          allocate(setFields(10))
+          allocate(setFields(22))
           setFields(1)  = h5_dataField(name="ID",     type=h5_typeInt)
           setFields(2)  = h5_dataField(name="TURN",   type=h5_typeInt)
-          setFields(3)  = h5_dataField(name="S",      type=h5_typeReal)
-          setFields(4)  = h5_dataField(name="X",      type=h5_typeReal)
-          setFields(5)  = h5_dataField(name="XP",     type=h5_typeReal)
-          setFields(6)  = h5_dataField(name="Y",      type=h5_typeReal)
-          setFields(7)  = h5_dataField(name="YP",     type=h5_typeReal)
-          setFields(8)  = h5_dataField(name="dE/E",   type=h5_typeReal)
-          setFields(9)  = h5_dataField(name="Z",      type=h5_typeReal)
-          setFields(10) = h5_dataField(name="KTRACK", type=h5_typeInt)
-          setFields(11)  = h5_dataField(name="E",      type=h5_typeReal)
-          setFields(12)  = h5_dataField(name="PC",      type=h5_typeReal)
-          setFields(13)  = h5_dataField(name="P/P0",     type=h5_typeReal)
-          setFields(14)  = h5_dataField(name="P0/P)",      type=h5_typeReal)
-          setFields(15)  = h5_dataField(name="BETA0/BETA",     type=h5_typeReal)
-          setFields(16)  = h5_dataField(name="MASS",   type=h5_typeReal)
-          setFields(17)  = h5_dataField(name="M/M0/Q0/Q", type=h5_typeReal)
-          setFields(18)  = h5_dataField(name="ENERGY0",      type=h5_typeReal)
-          setFields(19)  = h5_dataField(name="PC0",      type=h5_typeReal)
-          call h5_createFormat("dumpFormat3", setFields, dump_hdf5Format(3))
+          setFields(3)  = h5_dataField(name="KTRACK", type=h5_typeInt)
+          setFields(4)  = h5_dataField(name="S",      type=h5_typeReal)
+          setFields(5)  = h5_dataField(name="X",      type=h5_typeReal)
+          setFields(6)  = h5_dataField(name="XP",     type=h5_typeReal)
+          setFields(7)  = h5_dataField(name="Y",      type=h5_typeReal)
+          setFields(8)  = h5_dataField(name="YP",     type=h5_typeReal)
+          setFields(9)  = h5_dataField(name="SIGMA",  type=h5_typeReal)
+          setFields(10) = h5_dataField(name="(E-E0)/E0",   type=h5_typeReal)
+          setFields(11) = h5_dataField(name="E",      type=h5_typeReal)
+          setFields(12) = h5_dataField(name="PC",     type=h5_typeReal)
+          setFields(13) = h5_dataField(name="BETA0/BETA",     type=h5_typeReal)
+          setFields(14) = h5_dataField(name="(P-P0)/P0",   type=h5_typeReal)
+          setFields(14) = h5_dataField(name="1000*(P-P0)/P0",   type=h5_typeReal)
+          setFields(15) = h5_dataField(name="P0/P",   type=h5_typeReal)
+          setFields(16) = h5_dataField(name="MASS",   type=h5_typeReal)
+          setFields(17) = h5_dataField(name="CHARGE", type=h5_typeReal)
+          setFields(18) = h5_dataField(name="(CHARGE/CHARGE0)*(MASS0/MASS)", type=h5_typeReal)
```

Comment:
I think this is too much for a column header label. They need to be long enough to describe what's in them, but since these labels are used when accessing the data itself, they should be compact and more user friendly.

In my PR #670 I'm adding support for dataset metadata, where such information can be added to string arrays `colNames` and `colUnits`.